clients/admin/src/account.rs

```diff
@@ -48,7 +51,21 @@ pub fn info(core: &Core, username: Option<String>, public_key: Option<String>) -
 
     let account_info = core.admin_get_account_info(identifier)?;
 
-    println!("{:#?}", account_info);
+    println!("{}", account_info.username);
+    println!("root: {}", account_info.root);
+    println!("payment_platform: {:#?}", account_info.payment_platform);
+
+    let last_active = Duration::milliseconds(
+        (SystemTime::now()
+            .duration_since(UNIX_EPOCH)
+            .unwrap()
+            .as_millis() as u64
+            - account_info.last_active) as i64,
+    )
+    .format_human()
+    .to_string();
+    println!("last_active: {}", last_active);
+    println!("usage: {}B", account_info.usage);
```

Comment:
Thoughts about using our human readable bytes function here? The number would be less precise but maybe we'd want "10.1MB" instead of "1014349B"